beekeeper-cleanup/src/main/java/com/expediagroup/beekeeper/cleanup/aws/S3Client.java

```diff
@@ -73,10 +74,21 @@ List<String> deleteObjects(String bucket, List<String> keys) {
       return Collections.emptyList();
     }
     if (!dryRunEnabled) {
-      keys.forEach(key -> log.info("Deleting: \"{}/{}\"", bucket, key));
-      DeleteObjectsRequest deleteObjectsRequest = new DeleteObjectsRequest(bucket)
-          .withKeys(keys.toArray(new String[] {}));
-      DeleteObjectsResult deleteObjectsResult = amazonS3.deleteObjects(deleteObjectsRequest);
+      log.info("Attempting to delete a total of {} objects, from [{}] to [{}]", keys.size(), keys.get(0), keys.get(keys.size()-1));
```

Comment:
You need to leave a space before and after `-`. You have some other cosmetic issues with this change too. Please check with the team and run this change through our formatter to ensure there are no issues.

Reply:
fixed space and checked style formatter